현정/3주차_1926.java

```diff
@@ -0,0 +1,97 @@
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.util.LinkedList;
+import java.util.Queue;
+import java.util.StringTokenizer;
+
+public class BJ_1926 {
+    static int[][] array;
+    static int[][] visit;
+    static int[] dist_x = {1, 0, -1, 0};
+    static int[] dist_y = {0, 1, 0, -1};
+    static int n;
+    static int m;
+    static int answer = 0;
+    static class Point{
+        int x; int y;
+        public Point(int x, int y){
+            this.x = x;
+            this.y = y;
+        }
+    }
+
+    static int bfs(Point start) {
+        //방문 배열에 현재 위치 1로 값 변경
+        visit[start.x][start.y] = 1;
+        Queue<Point> que = new LinkedList<>();
+        // 큐에 현재 위치 푸시
+        que.offer(start);
+        // 그림 길이 0으로 초기화
+        int cnt = 0;
+
+        // 큐 비워질 때까지 bfs 탐색
+        while (!que.isEmpty()) {
+            Point p = que.poll();
```

Comment:
```suggestion
            Point current = que.poll();
```
제안처럼 변수에 조금 더 의미가 담겨있으면 좋겠네요
[변수 표기법](https://blog.naver.com/nkind/222017581135)
변수 표기법 자체는 다양한데, 카멜 / 파스칼 / 스네이크 세개중에 적절하게 잘 쓰면 더 가독성이 좋아집니다